### ISSUE TITLE
'RNPaypal.h' file not found

### ISSUE THREAD

**Issue by lightninglu10** [smarkets/react-native-paypal#11]
My AppDelegate.m file is here:

Get this error when I try to build: 

```
AppDelegate.m:4:9: 'RNPaypal.h' file not found
```

```
// Copyright 2015-present 650 Industries. All rights reserved.

#import "AppDelegate.h"
#import "RNPaypal.h"

@implementation AppDelegate

// Put your app delegate methods here. Remember to also call methods from EXStandaloneAppDelegate superclass
// in order to keep Expo working. See example below.

- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
{
  [[RNPaypal sharedInstance] configure];
  return [super application:application didFinishLaunchingWithOptions:launchOptions];
}

- (BOOL)application:(UIApplication *)application openURL:(NSURL *)url
  sourceApplication:(NSString *)sourceApplication annotation:(id)annotation
{
  return [[RNPaypal sharedInstance] application:application openURL:url sourceApplication:sourceApplication annotation:annotation];
}
@end
```

**Comment by EmmanuelID** [comment]
> Mi archivo AppDelegate.m está aquí:
> 
> Recibe este error cuando intento construir:
> 
> ```
> AppDelegate.m:4:9: 'RNPaypal.h' file not found
> ```
> ```
> // Copyright 2015-present 650 Industries. All rights reserved.
> 
> #import "AppDelegate.h"
> #import "RNPaypal.h"
> 
> @implementation AppDelegate
> 
> // Put your app delegate methods here. Remember to also call methods from EXStandaloneAppDelegate superclass
> // in order to keep Expo working. See example below.
> 
> - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
> {
>   [[RNPaypal sharedInstance] configure];
>   return [super application:application didFinishLaunchingWithOptions:launchOptions];
> }
> 
> - (BOOL)application:(UIApplication *)application openURL:(NSURL *)url
>   sourceApplication:(NSString *)sourceApplication annotation:(id)annotation
> {
>   return [[RNPaypal sharedInstance] application:application openURL:url sourceApplication:sourceApplication annotation:annotation];
> }
> @end
> ```



> My AppDelegate.m file is here:
> 
> Get this error when I try to build:
> 
> ```
> AppDelegate.m:4:9: 'RNPaypal.h' file not found
> ```
> ```
> // Copyright 2015-present 650 Industries. All rights reserved.
> 
> #import "AppDelegate.h"
> #import "RNPaypal.h"
> 
> @implementation AppDelegate
> 
> // Put your app delegate methods here. Remember to also call methods from EXStandaloneAppDelegate superclass
> // in order to keep Expo working. See example below.
> 
> - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
> {
>   [[RNPaypal sharedInstance] configure];
>   return [super application:application didFinishLaunchingWithOptions:launchOptions];
> }
> 
> - (BOOL)application:(UIApplication *)application openURL:(NSURL *)url
>   sourceApplication:(NSString *)sourceApplication annotation:(id)annotation
> {
>   return [[RNPaypal sharedInstance] application:application openURL:url sourceApplication:sourceApplication annotation:annotation];
> }
> @end
> ```

I have the same problem, did you manage to solve it?

**Comment by lightninglu10** [comment]
Never solved, I think this package is broken.

**Comment by EmmanuelID** [comment]
Do you know any alternative?

**Comment by lightninglu10** [comment]
Nope, but I do believe Paypal has a web checkout flow that you can tap into. Haven't gotten around to working on it.

**Comment by EmmanuelID** [comment]
Would you implement it with a WebView or how would you do it?

**Comment by lightninglu10** [comment]
Yeah, open up the webview flow and do it from there.

**Comment by lightninglu10** [comment]
Looking around again, I found this package: https://github.com/taessina/react-native-paypal-wrapper

If you implement it can you let me know if it works?

**Comment by EmmanuelID** [comment]
I am using react-native-paypal-lib on android but it has errors in IOS

https://github.com/zeddz92/react-native-paypal

**Comment by ptdede** [comment]
Hi all! Please try add
```
$(SRCROOT)/../node_modules/react-native-paypal/ios
```
to 
target > Build Settings > search header > find Header Search Paths.
Or see picture
<img width="1218" alt="Screen Shot 2019-04-15 at 14 28 59" src="https://user-images.githubusercontent.com/3388701/56111665-602a7980-5f8b-11e9-910e-4353a86f34fa.png">

**Comment by riso** [comment]
Hi guys, sorry for the late reply. 

I think we might have some issue in the `project.pbxproj` config file, as I recall having to do something similar as the latest comment suggested to have react-native-paypal build correctly in our project.

I think that's the issue, and I was wondering if the workaround suggested by @ptdede is working for anyone else?

**Comment by keivanh** [comment]
Thank you @ptdede, it worked for me.

**Comment by riso** [comment]
Closed by #13
